modules/alloy/src/main/ts/ephox/alloy/ui/schema/TypeaheadSchema.ts

```diff
@@ -55,7 +55,7 @@ const schema = Fun.constant([
     Focusing, Representing, Streaming, Keying, Toggling, Coupling
   ]),
 
-  FieldSchema.customField('lazyTypeaheadComp', () => Cell(Optional.none)),
+  FieldSchema.customField('lazyTypeaheadComp', () => Cell(Optional.none())),
```

Comment:
NOTE to other reviewers: This is just a bug. It only surfaces when you clobber the EventBehaviour that does onAttached in Typeahead, so it only happens when the code is broken, but it does need to be fixed.